move_together_app/lib/core/models/trip.dart

```diff
@@ -32,6 +39,7 @@ class Trip {
       endDate: DateTime.parse(json['endDate']),
       participants: json['participants'].map<Participant>((e) => Participant.fromJson(e)).toList(),
       inviteCode: json['inviteCode'],
+      photoUrl: json['photoUrl'],
```

Comment:
```suggestion
      photoUrl: json['photoUrl'] == null ? 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Tour_Eiffel_Wikimedia_Commons.jpg/260px-Tour_Eiffel_Wikimedia_Commons.jpg': dotenv.env['API_ADDRESS'] + "/banner/" + json['photoUrl'],
```

Tu peux mettre l'image par défaut ici comme ca t'as pas a faire de if dans les composants

Reply:
au final je fais coté backend (avec une image stockée sur le server)